db/schema.rb

```diff
@@ -447,10 +447,12 @@
     t.string "logo_formatted_name"
     t.string "analytics_identifier"
     t.boolean "visual_editor"
+    t.integer "government_id"
```

Comment:
just wondering if it should be called government_override in pattern with `person_override` to make it explicit that it's an override and not a usual thing?

Reply:
I think the person override is that feature for speeches where you can put arbitrary text in when the speech was given by somebody without a profile on GOV.UK.

In our case the government does exist on GOV.UK, it's just not the default one. So I feel that the use case isn't identical, and I also wonder if not making it clear that the column contains an id might lead to confusion.